src/components/ResultDisplay.module.css

```diff
@@ -28,3 +30,20 @@
 	padding: 1rem;
 	overflow: auto;
 }
+
+.sortWidget {
+	background-color: transparent;
+	border-width: 0;
+	padding-left: 0.5em;
+	padding-right: 0.5em;
+	transition: transform 0.05s ease-out;
```

Comment:
> `0.05s`

So fast! Maybe, a fifth or fourth of a second?

Reply:
Ok! Updated to .25s